earth_enterprise/src/common/generic_utils.h

```diff
@@ -17,6 +17,9 @@
 #ifndef GEO_EARTH_ENTERPRISE_SRC_COMMON_GENERIC_UTILS_H_
 #define GEO_EARTH_ENTERPRISE_SRC_COMMON_GENERIC_UTILS_H_
 
+#include <string>
```

Comment:
These lines don't seem to want to go away.

Reply:
Sorry about that, didn't pay close enough attention. Taken care of.